runtime/doc/lsp.txt

```diff
@@ -1469,6 +1462,58 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (integer)
 
 
+==============================================================================
+Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
+
+get({opts})                                         *vim.lsp.inlay_hint.get()*
+    Get the list of inlay hints, (optionally) restricted by buffer, client, or
+    range. This method does not make an LSP request, but just queries locally
+    stored inlay hints. The returned inlay hints are unresolved.
+
+    Example usage: >lua
+        local hint = vim.lsp.inlay_hint.get({ bufnr = bufnr })[1]
+        resolved_hint = client.request(methods.inlayHint_resolve, hint.inlay_hint, callback, bufnr)
+        vim.lsp.util.apply_text_edits(resolved_hint.textEdits, bufnr, 'utf-8')
+        command = resolved_hint.label[1].command
+        vim.lsp.buf.execute_command({ command = command })
+<
+
+    Parameters: ~
+      • {opts}  vim.lsp.inlay_hint.get.opts ? Optional table which holds the following optional fields:
+                • bufnr (integer?): Only return inlay hints from the clients
+                  attached to the given buffer.
+                • filter (function?): Predicate used to filter clients.
+                  Receives a client as argument and must return a boolean.
+                  Clients matching the predicate are included. Example: >lua
+
+                     -- Don't request inlay hints from typescript-language-server
+                     vim.lsp.inlay_hint.get {
+                       filter = function(client) return client.name ~= "tsserver" end
+                     }
+<
+                • id (integer?): Only return inlay hints from the client with
+                  ID (client.id) matching this field.
+                • name (string?): Only return inlay hints from the client with
+                  name (client.name) matching this field.
+                • range (lsp.Range?): Only return inlay hints within this
+                  range, inclusive. `lsp.Range` is a table with `start` and
+                  `end` keys with type `lsp.Position`. `lsp.Position` is
+                  (0,0)-indexed.
+
+    Return: ~
+        vim.lsp.inlay_hint.get.resp []:
```

Comment:
Maybe I'm just slow but I think that adding something like "Table of objects with the following fields...".

Reply:
Nah that's a fine idea, added

(also, looks like the generator added a space here too lol)

Reply:
Damn the type generator just keeps loosing points ehh